tests/common/database_fixture.hpp

```diff
@@ -85,6 +85,7 @@ extern uint32_t ( STEEMIT_TESTING_GENESIS_TIMESTAMP );
         BOOST_##TL( "exception '" BOOST_STRINGIZE( E ) "' is expected, but no exception thrown" ); }    \
     catch( E const& ex ) {                                                                              \
         const fc::variant_object &props = ex.get_log().at(0).get_data();                                \
+        (void)props; /*avoid "unused var" spam. TODO: @smedvedev, make it used?*/                       \
```

Comment:
props object in this place unused. In case of error, props will be printed in err.to_detail_string(). It's better to remove this variable.